src/TestWebsites/MVC/Views/Home/Bmp.cshtml

```diff
@@ -0,0 +1,99 @@
+﻿@{
+    ViewData["Title"] = "Bmp";
+}
+<article>
```

Comment:
This is all using very old  [Responsive](http://responsivebp.com/) specific markup. We shouldn't need it.

Are there any new images in the diff? We don't want them.